nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationTypes/01_CardPayoutRequest_DestinationTypeToken.yaml

```
@@ -13,9 +13,10 @@ properties:
     pattern: ^(tok)_(\w{26})$
     maxLength: 30
     minLength: 30
-    description: The Checkout.com token (eg a card, wallet or token)
+    description: The Checkout.com token (e.g., a card token)
```

```suggestion
    description: The Checkout.com token (for example, a card token)
```
Reason for suggestion: screen readers can misread 'e.g.' as 'egg'. 🥚